static/js/popovers.js

```diff
@@ -837,7 +838,7 @@ function get_user_sidebar_popover_items() {
         return undefined;
     }
 
-    return $("li:not(.divider):visible > a", current_user_sidebar_popover.$tip);
+    return $("li:not(.divider):visible a", current_user_sidebar_popover.$tip);
```

Comment:
What's the story with this change?

Reply:
Three-dot menus `<a>` tag is inside `<span>` tag unlike other items in the user info popover which are just inside `<li>` tag and because of that this selector was not capturing three-dot menus `<a>` tag

Other user info popovers capture a three-dot menu as they are not specifying `<a>` tag like that above line.
https://github.com/zulip/zulip/blob/da8817889bbd024656b39db0c98f30b95bd4de82/static/js/popovers.js#L381-L393

Reply:
OK. The risk of course is that if there's somehow another link inside one of these places, it might create different unexpected behavior, but this is probably fine.